src/StakingNode.sol

```diff
@@ -418,7 +420,7 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
         // 4. podOwnerShares: Active shares in Eigenlayer, representing staked ETH
         int256 totalETHBalance =
             int256(withdrawnETH + unverifiedStakedETH + queuedSharesAmount)
-            + eigenPodManager.podOwnerShares(address(this));
+            + eigenPodManager.podOwnerDepositShares(address(this));
```

Comment:
same concern here, needs to be investigated if podOwnerDepositShares is what can actually be withdrawn, what may actually be withdrawable could be less because of slashingFactor.